src/Coordination/KeeperServer.cpp

```diff
@@ -311,10 +312,25 @@ void KeeperServer::launchRaftServer(bool enable_ipv6)
 
     nuraft::ptr<nuraft::logger> logger = nuraft::cs_new<LoggerWrapper>("RaftInstance", coordination_settings->raft_logs_level);
     asio_service = nuraft::cs_new<nuraft::asio_service>(asio_opts, logger);
-    asio_listener = asio_service->create_rpc_listener(state_manager->getPort(), logger, enable_ipv6);
 
-    if (!asio_listener)
-        return;
+    std::vector<std::string> listen_hosts = DB::getMultipleValuesFromConfig(config, "", "interserver_listen_host");
```

Comment:
Probably some comment would be nice to have here that we're using the same network interfaces for keeper as for clickhouse itself